modules/installation-guide/partials/proc_installing-che-on-openshift-4-using-the-cli.adoc

```diff
@@ -0,0 +1,49 @@
+// installing-che-on-openshift-using-the-operator
+
+[id="installing-{prod-id-short}-on-openshift-4-using-the-cli_{context}"]
+= Installing {prod-short} on OpenShift 4 using the CLI
+
+.Prerequisites
+
+* `oc` is installed on the cluster. See link:https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html#installing-the-cli[Installing the CLI].
+
+* `{prod-cli}` is installed on the cluster. See xref:using-the-chectl-management-tool.adoc[].
+
+* __<namespace>__ represents the project of the target installation.
+
+.Procedure
+
+. Log in to OpenShift. See link:https://docs.openshift.com/container-platform/4.6/cli_reference/openshift_cli/getting-started-cli.html#cli-logging-in_cli-developer-commands[Logging in to the CLI].
++
```

Comment:
Instead of having fixed version of OCP in the link we can use `latest`, for instance:
`https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html#cli-logging-in_cli-developer-commands`

Reply:
This step is a prerequisite, it should move in the accurate section. Maybe refine "`oc` is available." to precise it should have same version as the target OpenShift cluster.